src/transforms/serializer.js

```diff
@@ -15,8 +15,12 @@ const protocols = {}
 function createProtocol (state, direction, version, customPackets, compiled = true) {
   const key = state + ';' + direction + ';' + version + (compiled ? ';c' : '')
   if (protocols[key]) { return protocols[key] }
+  
   const mcData = require('minecraft-data')(version)
-
+  if (mcData === null) {
+    throw new Error(`minecraft-data is null for version ${version}`)
```

Comment:
I think the error should be something like "no data available for version X"

Reply:
also the `=== null` won't match if it returns undefined, a simple if would work

Reply:
Minecraft data returns null that's fine https://github.com/PrismarineJS/node-minecraft-data/blob/master/index.js#L33

Better not rely on implicit cast if we can